jobbergate-slurp/slurp/migrators/job_scripts.py

```diff
@@ -2,61 +2,119 @@
-
-    :returns: An dict mapping legacy job_script ids to nextgen job_script ids
     """
-    job_scripts_map = {}
     logger.info("Migrating job_scripts to nextgen database")
 
-    s3man.nextgen.engine.ensure_bucket()
+    for job_script_batch in batch(legacy_job_scripts, batch_size):
 
-    for job_script in legacy_job_scripts:
-        owner_email = user_map[job_script["job_script_owner_id"]]["email"]
-        nextgen_application_id = application_map[job_script["application_id"]]
+        mogrified_params = ",".join(
+            [
+                nextgen_db.mogrify(
+                    """
+                    (
+                        %(id)s,
+                        %(name)s,
+                        %(description)s,
+                        %(owner_email)s,
+                        %(application_id)s,
+                        %(created)s,
+                        %(updated)s
+                    )
+                    """,
+                    dict(
+                        id=job_script["id"],
+                        name=job_script["job_script_name"],
+                        description=job_script["job_script_description"],
+                        owner_email=user_map[job_script["job_script_owner_id"]]["email"],
+                        application_id=job_script["application_id"],
+                        created=job_script["created_at"],
+                        updated=job_script["updated_at"],
+                    ),
+                )
+                for job_script in job_script_batch
+            ]
+        )
 
-        result = nextgen_db.execute(
+        nextgen_db.execute(
             """
             insert into job_scripts (
+                id,
                 job_script_name,
                 job_script_description,
                 job_script_owner_email,
                 application_id,
                 created_at,
                 updated_at
             )
-            values (
-                %(name)s,
-                %(description)s,
-                %(owner_email)s,
-                %(application_id)s,
-                %(created)s,
-                %(updated)s
-            )
-            returning id
-            """,
-            dict(
-                name=job_script["job_script_name"],
-                description=job_script["job_script_description"],
-                owner_email=owner_email,
-                application_id=nextgen_application_id,
-                created=job_script["created_at"],
-                updated=job_script["updated_at"],
+            values {}
+            """.format(
+                mogrified_params
             ),
         )
 
-        nextgen_jobscript_id = result.fetchone()["id"]
-        job_scripts_map[job_script["id"]] = nextgen_jobscript_id
+    logger.success("Finished migrating job_scripts")
+
+
+async def transfer_job_script_files(legacy_job_scripts):
+    """
+    Transfer job-script files from a column at the legacy database to nextgen s3.
```

Comment:
```suggestion
    Transfer job-script files from a column in the legacy database to nextgen s3.
```

Reply:
solved in [ec1a607](https://github.com/omnivector-solutions/jobbergate/pull/181/commits/ec1a6078ac96b075160525f22303b719aebc9425)